jsettlers.logic/src/main/java/jsettlers/logic/movable/simplebehaviortree/IBooleanConditionFunction.java

```diff
@@ -0,0 +1,7 @@
+package jsettlers.logic.movable.simplebehaviortree;
+
+import java.io.Serializable;
+import java.util.function.Function;
```

Comment:
We have to use the **java8**.* package to be able to run the code on Android (the java8 package comes from the streamsupport library and offers a backport of the Java8 possibilities). The same applies for the @FunctionalInterface. We need to explicitly import it with `import java8.lang.FunctionalInterface;`